examples/routing-v1/client/README.md

```diff
@@ -0,0 +1,68 @@
+# Delegated Routing V1 Command Line Client
+
+This is an example of how to use the Delegated Routing V1 HTTP client from Boxo.
+In this package, we build a small command line tool that allows you to connect to
+a Routing V1 endpoint and fetch content providers, peer information, as well as
+IPNS records for a certain IPNS name.
+
+## Build
+
+```bash
+> go build -o routing-client
+```
```

Comment:
@lidel that will not work:

```
The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
```

We'd have to remove the replace from the `go.mod` and ensure we'd keep it up to date with latest Boxo. Could be done with CI/release-process.